mailscanner/conf.php.example

```diff
@@ -39,6 +39,11 @@ define('DEBUG', false);
 // Define language (de, en, fr, it, nl, pt_br);
 define('LANG', 'en');
 
+//Session Handling - conflicts can exist when the your environment makes use of multiple php sessions on the same server
+// to resolve this, uncomment the following option.  See https://github.com/mailwatch/MailWatch/issues/730 for more info
+//define('SESSION_NAME', 'MailWatch');
+
+
```

Comment:
I just thought I'd reference it as it possibly better explains when you might need to use it, and also why we've done it in the first place, but happy to remove if wished.